### DISCUSSION TITLE
Users cannot login: Terms of Service Buttons (e.g. I accept) not visible in "light mode"

### DISCUSSION THREAD

**Discussion by dirkpetersen** [danny-avila/LibreChat#4167] [Q&A]
As my system defaults to dark mode, I never ran into this but all my users are stuck at first login as the Buttons "I do not accept" and "I accept" are not visible unless you hover over them, for example using Edge 129:

![image](https://github.com/user-attachments/assets/e0eee307-b8f1-49a8-b536-4f0c7637e5e3)

when using dark mode they are visible: 

![image](https://github.com/user-attachments/assets/346f2eae-bb95-4ada-a7f5-ed37f02737db)

Is there a workaround, .e.g. can I enforce "dark mode" ?

**Comment by danny-avila** [comment]
Fixed here! https://github.com/danny-avila/LibreChat/pull/4169

Thanks for the report
